routes/movies.routes.js

```diff
@@ -0,0 +1,92 @@
+const router = require("express").Router();
+const Celebrity = require("../models/Celebrity.model");
+const Movie = require("../models/Movie.model");
+
+router.get("/movies/create", (req, res, next) => {
+  res.render("movies/new-movie");
```

Comment:
Here, you were supposed to find the celebrities. 

```
router.get("/movies/create", async (req, res, next) => {
  try {
    const celebs = await Celebrity.find();
    res.render("movies/new-movie", {celebs});
  } catch(err) {
    console.log(err)
  }
});
```

Reply:
Just like you did when you started your celebrities routes, but now render the movies/new-movie view